client/coral-embed-stream/src/graphql/index.js

```diff
@@ -200,7 +200,11 @@ const extension = {
       variables: {id, edit},
     }) => ({
       updateQueries: {
-        EmbedQuery: (previousData, {mutationResult: {data: {editComment: {comment: {status}}}}}) => {
+        EmbedQuery: (previousData, {mutationResult: {data: {editComment: {comment, errors}}}}) => {
+          if (errors && errors.length) {
```

Comment:
We're going to streamline error handling soon, that makes this check here unnecessary.
Can you add a todo to revisit this after streamlining error handling?
You can also remove this check for now.